apps/ledger-live-mobile/e2e/specs/market.spec.ts

```diff
@@ -0,0 +1,64 @@
+import * as detox from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import MarketPage from "../models/market/marketPage";
+import { loadConfig } from "../bridge/server";
+import { isAndroid, getElementByText } from "../helpers";
+
+let portfolioPage: PortfolioPage;
+let marketPage: MarketPage;
+
+describe("Market page for user with no device", () => {
+  beforeAll(async () => {
+    await loadConfig("1accountEth", true);
```

Comment:
you can remove the `await` here too. `loadConfig` isn't async. You can check the return type is `void` which means it just does something and doesn't return anything 
![image](https://github.com/LedgerHQ/ledger-live/assets/87659842/d21bf7d1-cbc6-4211-ae8f-199ecb2e044b)

Async functions return a `Promise<type>`, for example: 
![image](https://github.com/LedgerHQ/ledger-live/assets/87659842/333e90fc-f9a6-4a26-8903-2166ab76eca7)